Dockerfile.vpn

```diff
@@ -0,0 +1,8 @@
+# Use a Microsoft-approved image
+FROM mcr.microsoft.com/azure-cli:2.61.0 AS base
```

Comment:
Updated the `env.example`

Reply:
Thanks for the suggestion!

I did try several slimmer images (including Mariner), but I ran into issues with missing dependencies needed to run the VPN properly. Unfortunately, none of them worked without breaking something.

Also, when we used the Red Hat registry image in the pipeline, we got security warnings from Microsoft, so we had to avoid that.

For now, the current image works without issues or security concerns, but I’m open to exploring better options if we find something.